editor/script_editor_debugger.cpp

```diff
@@ -985,33 +989,23 @@ void ScriptEditorDebugger::_notification(int p_what) {
 		case NOTIFICATION_PROCESS: {
 
 			if (connection.is_valid()) {
-				inspect_scene_tree_timeout -= get_process_delta_time();
-				if (inspect_scene_tree_timeout < 0) {
-					inspect_scene_tree_timeout = EditorSettings::get_singleton()->get("debugger/scene_tree_refresh_interval");
-					if (inspect_scene_tree->is_visible()) {
-						_scene_tree_request();
-
-						if (inspected_object_id != 0) {
-							//take the chance and re-inspect selected object
-							Array msg;
-							msg.push_back("inspect_object");
-							msg.push_back(inspected_object_id);
-							ppeer->put_var(msg);
-							inspected_object_id = 0;
-						}
-					}
-				}
-
 				inspect_edited_object_timeout -= get_process_delta_time();
 				if (inspect_edited_object_timeout < 0) {
 					inspect_edited_object_timeout = EditorSettings::get_singleton()->get("debugger/remote_inspect_refresh_interval");
-					if (inspect_scene_tree->is_visible() && inspected_object_id) {
-						//take the chance and re-inspect selected object
+
+					if (inspect_scene_tree->is_visible())
+						_scene_tree_request();
+
+					bool need_query_instance = inspected_object_id != 0;
```

Comment:
I guess you could simplify this a bit as:
```
bool need_query_instance =
		inspected_object_id != 0
		&& editor->get_property_editor()->is_visible()
		&& editor->get_property_editor()->obj != NULL
		&& editor->get_property_editor()->obj->is_type("ScriptEditorDebuggerInspectedObject")
		&& (ObjectID)editor->get_property_editor()->obj->call("get_remote_object_id") != 0;
```
Or whatever format `clang-format` will allow for line breaks, it's a bit touchy with those. Worst case, everything on one line is still more readable than 5 successive assignments IMO.

Reply:
Done with clang-format :)